apps/nextjs-app/next.config.js

```diff
@@ -336,6 +340,7 @@ if (process.env.ANALYZE === 'true') {
   const withBundleAnalyzer = require('@next/bundle-analyzer')({
     enabled: true,
   });
+  // @ts-ignore
```

Comment:
was this really needed?

Reply:
The config is some how mistyped. Didn't want to use time to that now.